notifications.yaml

```diff
@@ -0,0 +1,268 @@
+swagger: '2.0'
+info:
+  version: 0.0.1
+  title: Notification API
+  description: Notification API.
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/messages/{id}':
+    parameters:
+      - name: id
+        in: path
+        type: string
+        required: true
+    get:
+      operationId: getMessage
+      summary: Get Message
+      description: Returns a previously submitted message.
+      responses:
+        '200':
+          description: Message found.
+          schema:
+            $ref: '#/definitions/AcceptedMessage'
+        '404':
+          description: No message found.
+  '/users/{fiscal_number}/messages':
+    parameters:
+      - name: fiscal_number
+        in: path
+        type: string
+        required: true
+        description: The fiscal number of the user.
+        pattern: >-
+          [A-Z]{6}[0-9LMNPQRSTUV]{2}[ABCDEHLMPRST][0-9LMNPQRSTUV]{2}[A-Z][0-9LMNPQRSTUV]{3}[A-Z]
+        maxLength: 16
+        minLength: 1
+      - $ref: '#/parameters/PaginationRequest'
+    get:
+      operationId: getMessagesByUser
+      summary: Get messages by user
+      description: |-
+        Returns the submitted messages the have the user as destination.
+        Messages will be returned in inverse delivery order (from last to first)
+      responses:
+        '200':
+          description: Found.
+          schema:
+            allOf:
+              - type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/definitions/AcceptedMessage'
+              - $ref: '#/definitions/PaginationResponse'
+          examples: {}
+        '404':
+          description: No user found.
```

Comment:
There are no standard and/or common view about it:
https://www.google.it/search?q=http+status+code+for+empty+result

I prefer the 200OK semantic for empty collections as stating that a collection is empty IMHO differs from stating that the collection does not exists at all (because there's no user associated).